db/migrate/20170221225907_create_items.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :items do |t|
       t.string :title
       t.string :description
-      t.decimal :price, :precision => 2
+      t.float :price 
```

Comment:
Yeah, I couldn't get decimal to work properly. We can do a new migration if we figure out a better strategy for storing price down the line.